src/v2i-hub/SpatPlugin/src/SpatPlugin.cpp

```diff
@@ -63,7 +63,7 @@ namespace SpatPlugin {
 				scConnection = std::make_unique<SignalControllerConnection>(ip_address, port, signal_group_mapping_json, signal_controller_ip, signal_controller_snmp_port,signal_controller_snmp_community, intersection_name, intersection_id);
 			}
 			// Only send SNMP Rquest to enable spat in simulation mode and binary spat. TFHRC TSCs do not expose this OID so calls to it will fail in hardware deployment
-			auto connected = scConnection->initializeSignalControllerConnection(PluginClientClockAware::isSimulationMode() && spatMode == "BINARY" );
+			auto connected = scConnection->initializeSignalControllerConnection(false);
```

Comment:
I could also add a new configuration parameter that controls whether to send this SNMP Request. Maybe that is cleaner that always setting this to false, which begs the question why do we have it as an argument at all.